src/anemoi/training/config/training/default.yaml

```diff
@@ -33,10 +33,34 @@ swa:
 # use ZeroRedundancyOptimizer ; saves memory for larger models
 zero_optimizer: False
 
-# dynamic rescaling of the loss gradient
-# see https://arxiv.org/pdf/2306.06079.pdf, section 4.3.2
-# don't enable this by default until it's been tested and proven beneficial
-loss_gradient_scaling: False
+# loss functions
+loss_functions:
```

Comment:
This needs a better name as there's only one loss in here.

Reply:
Do you have a suggestion?
As this configures the losses, and metrics it made sense to me

Reply:
how about removing this higher level, so we have training.loss and training.metrics?